### PR TITLE
[RHACS][Docs][4.6 and later] ROX-26680: ARM support on secured clusters

### PR DESCRIPTION
**Pull request by agantony**
Version(s):
4.6+

Issue:
https://issues.redhat.com/browse/ROX-26680

Link to docs preview:
https://84962--ocpdocs-pr.netlify.app/openshift-acs/latest/installing/acs-high-level-overview.html
https://84962--ocpdocs-pr.netlify.app/openshift-acs/latest/installing/installing_other/install-secured-cluster-other.html

SME review:
- [x] SME has approved this change.

Additional information:
- Merge into `rhacs-docs-main` and cherry-pick to:
    - `rhacs-docs-4.6`

### CODE REVIEW

File: installing/installing_other/install-secured-cluster-other.adoc [hunk 1]
```diff
@@ -8,7 +8,12 @@ toc::[]
 
 [role="_abstract"]
 
-You can install {product-title-short} on your secured clusters for platforms such as Amazon Elastic Kubernetes Service (Amazon EKS), Google Kubernetes Engine (Google GKE), and Microsoft Azure Kubernetes Service (Microsoft AKS).
+You can install {rh-rhacs-first} on your secured clusters for the following platforms:
+
+* Amazon Elastic Kubernetes Service (Amazon EKS)
+* Google Kubernetes Engine (Google GKE)
+* Microsoft Azure Kubernetes Service (Microsoft AKS)
+* Azure Resource Manager (ARM)
```
Comment (Molter73) [inline installing/installing_other/install-secured-cluster-other.adoc:16]: This is not a k8s platform, when we say Arm support, we mean the [Arm architecture](https://www.arm.com/architecture), similar to Power (ppc64le architecture) and Z (s390x architecture) are also supported.

File: modules/installation-methods-for-different-architectures.adoc [hunk 2]
```diff
@@ -14,10 +14,13 @@
...
 |AMD64
-a|Operator (preferred), Helm charts, or `roxctl` CLI (not recommended)
+a|Operator (preferred), Helm charts, `roxctl` CLI (not recommended)
 
 | ppc64le ({ibm-power})
 .2+a|Operator
 
 | s390x ({ibm-z} and {ibm-linuxone})
+
+|Azure Resource Manager (ARM)
```
Comment (Molter73) [inline modules/installation-methods-for-different-architectures.adoc:24]:
```suggestion
|aarch64 ({arm64})
```

File: modules/installation-methods-for-different-architectures.adoc [hunk 3]
```diff
@@ -14,10 +14,13 @@
...
+|Helm charts (recommended), `roxctl` CLI
```
Comment (Molter73) [inline modules/installation-methods-for-different-architectures.adoc:25]: We probably want Operator here as well, I just didn't have time to test it, but it is supported.

File: installing/installing_other/install-secured-cluster-other.adoc [hunk 4]
```diff
@@ -8,7 +8,11 @@ toc::[]
 
 [role="_abstract"]
 
-You can install {product-title-short} on your secured clusters for platforms such as Amazon Elastic Kubernetes Service (Amazon EKS), Google Kubernetes Engine (Google GKE), and Microsoft Azure Kubernetes Service (Microsoft AKS).
+You can install {rh-rhacs-first} on your secured clusters for the following platforms:
+
+* Amazon Elastic Kubernetes Service (Amazon EKS)
+* Google Kubernetes Engine (Google GKE)
```
Comment (jeana-redhat) [inline installing/installing_other/install-secured-cluster-other.adoc:14]:
```suggestion
* Google Kubernetes Engine (GKE)
```

File: modules/installation-methods-for-different-architectures.adoc [hunk 5]
```diff
@@ -14,10 +14,13 @@
...
-a|Operator (preferred), Helm charts, or `roxctl` CLI (not recommended)
+a|Operator (preferred), Helm charts, `roxctl` CLI (not recommended)
 
 | ppc64le ({ibm-power})
 .2+a|Operator
 
 | s390x ({ibm-z} and {ibm-linuxone})
+
+|aarch64 ({arm64})
+|Operator (preferred), Helm charts, `roxctl` CLI
```
Comment (jeana-redhat) [inline modules/installation-methods-for-different-architectures.adoc:25]: [The table immediately above this](https://84962--ocpdocs-pr.netlify.app/openshift-acs/latest/installing/acs-high-level-overview.html#install-platforms-methods_acs-high-level-overview) uses "or", and that matches standard grammar for a list format. 

```suggestion
a|Operator (preferred), Helm charts, or `roxctl` CLI (not recommended)

| ppc64le ({ibm-power})
.2+a|Operator

| s390x ({ibm-z} and {ibm-linuxone})

|aarch64 ({arm64})
|Operator (preferred), Helm charts, or `roxctl` CLI
```

**Review by jeana-redhat — COMMENTED** [review]
A couple small things here

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Review by Molter73 — APPROVED** [review]

**Pull request merged**